spec.bs

```diff
@@ -232,7 +232,105 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<xmp class="idl">
+[Exposed=Window, Global=Window]
+interface InterestGroupBiddingScriptRunnerGlobalScope{
+};
+
+[Exposed=Window, Global=Window]
+interface InterestGroupScoringScriptRunnerGlobalScope{
+};
+
+[Exposed=Window, Global=Window]
+interface InterestGroupReportingScriptRunnerGlobalScope{
+};
+</xmp>
+
+Issue: where do these interfaces above get exposed to?
+
+TODO: modelingSignals
+
+<xmp class="idl">
+[Exposed=InterestGroupBiddingScriptRunnerGlobalScope]
+dictionary AdSize {

```

Comment:
Dictionaries do not get "exposed" as they are not actual web-exposed types. They just define the shape of objects that get passed into APIs basically. Exposed is an invalid extended attribute to apply here.